src/Classes/ServiceAPI/MyRadio_Swagger2.php

```diff
@@ -348,6 +370,44 @@ private static function getTags($apis)
         return $tags;
     }
 
+    private static function getMethodOpType($method)
+    {
+        $name = $method->getName();
+
+        //Note the ordering is important - create is static!
+        if (
+            CoreUtils::startsWith($name, 'create') ||
+            CoreUtils::startsWith($name, 'add')
+        ) {
```

Comment:
Except the build will fail if there's more than 120 chars per line. I suspect this one would still be under, but I felt it made sense for both ifs in this method to be consistent.